frontend/public/sitemap.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:image="http://www.google.com/schemas/sitemap-image/1.1">
+  <url>
+    <loc>https://form.gov.sg/</loc>
+    <priority>1.0</priority>
+    <image:image>
+       <image:loc>
+         https://form.gov.sg/static/images/landing/restricted__build-demo.png
```

Comment:
I carried that image from angular, but actually the screenshot is wrong, we'll need to take another @staceytan1998 🤔